slidez-fe-core/src/pages/dashboard/edit-profile/update.scss

```diff
@@ -0,0 +1,228 @@
+@import "src/global/global";
+
+.user-profile {
+    display: flex;
+    align-items: center;
+    .avatar {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 43px;
+        height: 43px;
+        top: calc(50% - 45px / 2 - 554px);
```

Comment:
What is the purpose of this property? It works only with position: absolute/relative/fixed

Reply:
it's not needed, I deleted this. Thanks)